proto/footprints/v1/footprints_service.proto

```
@@ -0,0 +1,54 @@
+syntax = "proto3";
+
+package footprints.v1;
+
+import "google/protobuf/timestamp.proto";
+import "profile/v1/service.proto";
```

Buf (1.70.0)

[error] imported file does not exist

(COMPILE)